django-prosoul/prosoul/panels/templates/attribute-template.json

```diff
@@ -1,23 +1,24 @@
 {
     "dashboard": {
-        "id": "AWG5dj8SETw-8I6Z4dX-",
+        "id": "activity_docker-smells",
         "value": {
             "description": "",
             "hits": 0,
             "kibanaSavedObjectMeta": {
-                "searchSourceJSON": "{\"filter\":[{\"$state\":{\"store\":\"appState\"},\"meta\":{\"alias\":\"Commits+Bugs\",\"disabled\":false,\"index\":\"ea1313b0-8f29-11e8-a362-0d37aacd7dee\",\"key\":\"query\",\"negate\":false,\"type\":\"custom\",\"value\":\"{\\\"bool\\\":{\\\"should\\\":[{\\\"term\\\":{\\\"metric_name\\\":\\\"numberOfBugs\\\"}},{\\\"term\\\":{\\\"metric_name\\\":\\\"https://github.com/grimoirelab/perceval.git_numberOfCommits\\\"}},{\\\"term\\\":{\\\"metric_name\\\":\\\"https://github.com/grimoirelab/GrimoireELK.git_numberOfCommits\\\"}}]}}\"},\"query\":{\"bool\":{\"should\":[{\"term\":{\"metric_name\":\"numberOfBugs\"}},{\"term\":{\"metric_name\":\"https://github.com/grimoirelab/perceval.git_numberOfCommits\"}},{\"term\":{\"metric_name\":\"https://github.com/grimoirelab/GrimoireELK.git_numberOfCommits\"}}]}}},{\"query\":{\"match_all\":{}}}],\"highlightAll\":true,\"version\":true}"
+                "searchSourceJSON": "{\"filter\": [{\"$state\": {\"store\": \"appState\"}, \"meta\": {\"alias\": null, \"disabled\": false, \"index\": \"ossmeter\", \"key\": \"metric_name\", \"negate\": false, \"params\": [\"Sum of Docker Smells\", \"Sum of Docker Dependencies\"], \"type\": \"phrases\", \"value\": \"Sum of Docker Smells, Sum of Docker Dependencies\"}, \"query\": {\"bool\": {\"minimum_should_match\": 1, \"should\": [{\"match_phrase\": {\"metric_name\": \"Sum of Docker Smells\"}}, {\"match_phrase\": {\"metric_name\": \"Sum of Docker Dependencies\"}}]}}}, {\"query\": {\"match_all\": {}}}], \"highlightAll\": true, \"version\": true}"
```

Comment:
same as the above comment, :thinking:

Reply:
That's strange, the new index value is an old one. It should be `ea1313b0-8f29-11e8-a362-0d37aacd7dee`

I'm wondering if the attribute-template.json is used.

Reply:
Oh, anything wrong with this change? :thinking:

Reply:
It seems so.

Can you try to make a test with the previous version and see what happens? thanks

Reply:
Hi @valeriocos 
I reverted only this change (removed this line and added the previous line) and I performed a visualization again (deleted the dashboard too before viz) and it worked. The dashboard was generated normally.

I think we can revert this change, WDYT?

Reply:
perfect, thanks @vchrombie!